src/content/reference/react/PureComponent.md

```diff
@@ -26,11 +26,10 @@ class Greeting extends PureComponent {
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `PureComponent` {/*purecomponent*/}
-
-To skip re-rendering a class component for same props and state, extend `PureComponent` instead of [`Component`:](/reference/react/Component)
+Untuk melewatkan render ulang sebuah komponen kelas untuk props dan state yang sama, perluas `PureComponent`, bukan [`Component`:](/reference/react/Component)
```

Comment:
Missing new line after the heading

Reply:
extend in this refer to `extends` in class component.
I suggest to keep as it is, using italic font.
Or we can introduce for the first time by adding the original word.

E.g: `memperluas (*extend*)`

Reply:
I think using `*extends*` is more relevant somehow

```suggestion
Untuk melewatkan render ulang sebuah komponen kelas untuk props dan state yang sama, *extends* `PureComponent`, bukan [`Component`:](/reference/react/Component)
```